spec/openapi.yaml

```diff
@@ -24,11 +26,17 @@ paths:
         - Team Info
       responses:
         '200':
-          description: Successful response
+          description: Successfully found the team info.
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/TeamInfoResult'
+        '401':
+          description: A valid API key is required to access this resource.
+        '404':
+          description: Team not found.
```

Comment:
To avoid team enumeration, implementation should also return 404 if team exists but API key doesn't grant access to the team.